source/includes/api/_before_you_get_started.md

```diff
@@ -0,0 +1,17 @@
+# Before you get started
+
+This section covers a list of topics you should be familiar with before using the API. These concepts span the whole API.
+
+## Applications
+
+TODO
+
+
+## Authentication
+
+TODO
+
+
+## Roles
+
+TODO
```

Comment:
Might be worth adding **Environments** here too

Reply:
do you mean staging/production clusters, or the `environment` parameter that datasets, layers and so on have?

Reply:
the environment parameter that datasets, layers and so on have :)